core/src/test/java/tech/pegasys/eth2signer/core/signing/FilecoinAddressTest.java

```diff
@@ -56,7 +56,7 @@ void referenceTestsForActorAddresses(final String address, final String payload)
 
   @Test
   void addressWithInvalidChecksumThrowsError() {
-    assertThatThrownBy(() -> FilecoinAddress.decode("f17uoq6tp427uzv7fztkbsnn64iwotfrristwprz"))
+    assertThatThrownBy(() -> FilecoinAddress.decode("f17uoq6tp427uzv7fztkbsnn64iwotfrristwpr"))
```

Comment:
Had to change this value as this was actually an invalid length for base32 and we don't have any padding in filecoin addresses.